sdk-extensions/incubator/src/main/java/io/opentelemetry/sdk/extension/incubator/fileconfig/SpanExporterFactory.java

```diff
@@ -93,6 +93,29 @@ public SpanExporter create(
               "otlp exporter"));
     }
 
+    Zipkin zipkin = model.getZipkin();
+    if (zipkin != null) {
```

Comment:
Nit: the contents of this block could be extracted to a private method (same for OTLP). As we add more exporters here this is quickly going to get unwieldy